### DISCUSSION TITLE
Really dumb beginner question about using "sm:w-1/2"

### DISCUSSION THREAD

**Discussion by armenr** [unocss/unocss#1413] [Q&A] [answered]
### Simple Use-Case

Let's say I have this "generic" TailWaind code:

```
<img class="hidden md:block absolute top-0 right-0 w-36 sm:w-1/2 lg:w-auto lg:h-full object-contain" src="background-orange-right.svg" alt="">
```

And I'd rather that it look this way:
```
    <img
      hidden
      md:block
      absolute
      top-0
      right-0
      w-36
      sm:w-1/2
      lg:w-auto
      lg:h-full
      object-contain
      src="background-orange-right.svg" alt=""
    >
```

### Problem

Unfortunately, I'm a newb, so I am running into the following:
```
[plugin:vite-plugin-vue-inspector] Illegal '/' in tags.
```

And this is due to:
`      sm:w-1/2`

### Question:

What's the "right" way to do this with Uno? I must be missing something *really* painfully obvious.

Thanks!

**Comment by zyyv** [comment, marked as answer]
If you had use `presetAttributify`, you can use attribute mode to write class. 
But `/` doesn't conform to html attribute rules. You can use `class="sm:w-1/2"` or `sm="w-1/2"` instead of it to avoid errors.

**Reply by armenr** [reply]
Thanks @chris-zhu. 

I'm working off of @antfu 's excellent Vitesse boilerplate, so `presetAttributify` is enabled, out of the box...and my brain just blanked when I ran into this.

It's so obvious 🤣 now that I look at it.

Thanks for the kind reply!
